core/__tests__/models/destination/plugins/exportRecord.ts

```diff
@@ -955,6 +955,72 @@ describe("models/destination - with custom exportRecord plugin", () => {
       });
     });
 
+    describe("when destination delivery mode is", () => {
+      let record: GrouparooRecord = null;
```

Comment:
TS note: You don't need to set it to null.

```suggestion
      let record: GrouparooRecord;
```

This makes the record start with a value of `undefiend`